sugargame/canvas.py

```diff
@@ -4,7 +4,7 @@
 from gi.repository import GLib
 from sugar3.activity.activity import PREVIEW_SIZE
 import pygame
-import event
+import sugargame.event as event
```

Comment:
Would prefer an upgrade of Sugargame (the code is over in the Sugargame repository) to be in a separate commit.  Also, I see you have not updated the Sugargame version.  Perhaps you didn't reference the repository.

Reply:
How reference can be done, I am unsure?

Reply:
Sorry, I don't understand.  Please try again.

Reply:
The last part of your first message "Perhaps you didn't refernce the repository" I am unsure how it is done.

Reply:
Sugargame is a software library maintained in another repository.  We make changes to Sugargame in that other repository, and then add those changes in one commit to each activity that adopts the new release of Sugargame.  Your commits here have changed Sugargame in this repository, rather than adopt a new release of Sugargame.  This is confusing, as it will potentially result in many different versions of Sugargame across the activity set.

Reply:
Thankyou ok will do that.